src/push.js

```diff
@@ -58,23 +67,31 @@ function sendWebPush(message, subscription, authToken) {
   // GCM servers support the Web Push protocol. This should go away in the
   // future.
   const endpoint = subscription.endpoint.replace(GCM_URL, TEMP_GCM_URL);
-
-  const payload = encrypt(message, subscription);
   const headers = {
-    'Content-Encoding': 'aesgcm',
-    'Encryption': `salt=${ub64(payload.salt)}`,
-    'Crypto-Key': `dh=${ub64(payload.serverPublicKey)}`
+    // TODO: Make TTL variable
+    Ttl: '0'
```

Comment:
Mega nit, shouldn't this be TTL similar to: https://github.com/gauntface/simple-push-demo/pull/28/files